### ISSUE TITLE
Backup & Restore between different OS - Unable to alter path in new OS

### ISSUE THREAD

**Issue by jerom18** [SickRage/SickRage#2488]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch: master
Commit: 21c9621a36dabe345fc4879200c3fba39b84c8ce
Version: 2016.10.11-1
Database Version: 44.0
Python Version:2.7.10 (default, May 23 2015, 09:44:00) [MSC v.1500 64 bit (AMD64)]
SSL Version: OpenSSL 1.0.2a 19 Mar 2015
OS:Windows-8-6.2.9200

What you did: Last week moved from Ubuntu 16:04 based server to Windows 10 server. I backed up settings on Ubuntu and restored on Windows, this initially appeared to work well. The Windows server has limited storage and has run Sickrage fine for a year or so. I simply restored the settings back into the previously working Sickrage instance. There are 3 instances of Sickrage on the Windows server and they have co-existed happily for about 6 months or so.  

What happened: Now I am not able to consistently post process from Sabnzb, it works sometimes but for now clear reason why. I am also not able to manually post process, this gives more information. The other 2 instances of Sickrage are able to post process and manual process without issue on all downloads.

What you expected: I expected Sabnzb to be able to post process as on Ubuntu and expect to be able to manually post process. I have checked the config and py files for reference to the path from Ubuntu but have not been able to locate it. I see this as less of an issue more need for a guide or something of how to move between OS if this is even supported. I see that I might be as well to delete all the config and start again I just don't want to lose all the program status.

Logs:

```
2016-10-14 20:04:04 Thread-19 :: [21c9621] Failed doing webui callback: Traceback (most recent call last):
  File "C:\SickRage\sickbeard\webserve.py", line 270, in async_call
    result = function(**kwargs)
  File "C:\SickRage\sickbeard\webserve.py", line 2322, in processEpisode
    is_priority=argToBool(is_priority), delete_on=argToBool(delete_on), failed=argToBool(failed), proc_type=proc_type
  File "C:\SickRage\sickbeard\processTV.py", line 288, in processDir
    process_media(processPath, videoFiles, nzbName, process_method, force, is_priority, result)
  File "C:\SickRage\sickbeard\processTV.py", line 558, in process_media
    result.result = processor.process()
  File "C:\SickRage\sickbeard\postProcessor.py", line 1051, in process
    if not verify_freespace(self.file_path, ep_obj.show._location, [ep_obj] + ep_obj.relatedEps, method=self.process_method):  # pylint: disable=protected-access
  File "C:\SickRage\sickbeard\helpers.py", line 1578, in verify_freespace
    if method == "move" and ek(os.stat, src).st_dev == ek(os.stat, dest if ek(os.path.exists, dest) else ek(os.path.dirname, dest)).st_dev:  # pylint: disable=no-member
  File "C:\SickRage\sickrage\helper\encoding.py", line 38, in ek
    result = function(*args, **kwargs)
WindowsError: [Error 3] The system cannot find the path specified: u'/home/TV'

Further Logging Debug

AAWindowsError: [Error 3] The system cannot find the path specified: u'/home/TV'
AA    result = function(*args, **kwargs)
AA  File "C:\SickRage\sickrage\helper\encoding.py", line 38, in ek
AA    if method == "move" and ek(os.stat, src).st_dev == ek(os.stat, dest if ek(os.path.exists, dest) else ek(os.path.dirname, dest)).st_dev:  # pylint: disable=no-member
AA  File "C:\SickRage\sickbeard\helpers.py", line 1578, in verify_freespace
AA    if not verify_freespace(self.file_path, ep_obj.show._location, [ep_obj] + ep_obj.relatedEps, method=self.process_method):  # pylint: disable=protected-access
AA  File "C:\SickRage\sickbeard\postProcessor.py", line 1051, in process
AA    result.result = processor.process()
AA  File "C:\SickRage\sickbeard\processTV.py", line 558, in process_media
AA    process_media(processPath, videoFiles, nzbName, process_method, force, is_priority, result)
AA  File "C:\SickRage\sickbeard\processTV.py", line 288, in processDir
AA    is_priority=argToBool(is_priority), delete_on=argToBool(delete_on), failed=argToBool(failed), proc_type=proc_type
AA  File "C:\SickRage\sickbeard\webserve.py", line 2322, in processEpisode
AA    result = function(**kwargs)
AA  File "C:\SickRage\sickbeard\webserve.py", line 270, in async_call
AATraceback (most recent call last):
AAWindowsError: [Error 3] The system cannot find the path specified: u'/home/TV'
AA    result = function(*args, **kwargs)
AA  File "C:\SickRage\sickrage\helper\encoding.py", line 38, in ek
AA    if method == "move" and ek(os.stat, src).st_dev == ek(os.stat, dest if ek(os.path.exists, dest) else ek(os.path.dirname, dest)).st_dev:  # pylint: disable=no-member
AA  File "C:\SickRage\sickbeard\helpers.py", line 1578, in verify_freespace
AA    if not verify_freespace(self.file_path, ep_obj.show._location, [ep_obj] + ep_obj.relatedEps, method=self.process_method):  # pylint: disable=protected-access
AA  File "C:\SickRage\sickbeard\postProcessor.py", line 1051, in process
AA    result.result = processor.process()
AA  File "C:\SickRage\sickbeard\processTV.py", line 558, in process_media
AA    process_media(processPath, videoFiles, nzbName, process_method, force, is_priority, result)
AA  File "C:\SickRage\sickbeard\processTV.py", line 288, in processDir
AA    is_priority=argToBool(is_priority), delete_on=argToBool(delete_on), failed=argToBool(failed), proc_type=proc_type
AA  File "C:\SickRage\sickbeard\webserve.py", line 2322, in processEpisode
AA    result = function(**kwargs)
AA  File "C:\SickRage\sickbeard\webserve.py", line 270, in async_call
2016-10-14 17:38:31 ERROR    Thread-20 :: [21c9621] Failed doing webui callback: Traceback (most recent call last):

SABnzb Script Log

[17:08:26] [INFO]::MAIN: #########################################################
[17:08:26] [INFO]::MAIN: ## ..::[nzbToMedia.py]::.. ##
[17:08:26] [INFO]::MAIN: #########################################################
[17:08:26] [INFO]::MAIN: Script triggered from SABnzbd
[17:08:28] [ERROR]::MAIN: Unable to open URL
[17:08:29] [INFO]::MAIN: Auto-detected SECTION:SickBeard
[17:08:29] [INFO]::MAIN: Calling SickBeard:tvja to post-process:American.Horror.Story.S06E05.720p.HDTV.x264-Obfuscated.nzb
[17:08:30] [INFO]::MAIN: SickBeard:tvja fork set to sickrage
[17:08:30] [INFO]::MAIN: FLATTEN: Flattening directory: C:\Users\Admin\Documents\Downloads\Download-TVS\Insert TV Show
[17:08:30] [POSTPROCESS]::SICKBEARD: SUCCESS: The download succeeded, sending a post-process request
[17:08:36] [ERROR]::SICKBEARD: Server returned status 500
[17:08:36] [ERROR]::MAIN: A problem was reported in the C:\nzbToMedia\nzbToSickBeard.py script.
SickBeard: Failed to post-process - Server returned status 500!
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by neoatomic** [comment]
Generally if you switch OS system then best to start fresh.

Quickly peeking trough the log i not the path `/home/TV` that looks like a linux folder instead of Windows.
Have you checked the root shows folder that you setup for Sickrage.? It might be pointing to your old Linux folder.

**Comment by jerom18** [comment]
The /home/TV path is the linux folder the issue I have is that I was not able to delete that as a root show path as Sickrage still believed that that is where the old TV shows resided. Even though I have changed them in Mass Update to the new windows path. I have since just removed the backup and started a fresh. The issue I had was not using Metadata and not wanting to lose the episode status for my library. I used Trakt to back that up though in the end. 

I tried both the above in Mass Update and in the config.ini but it wouldn't let me actually delete the root folder itself which was a pain. 

I guess its not an issue that often arises.
